components/strava/actions/create-activity/create-activity.mjs

```diff
@@ -0,0 +1,88 @@
+import strava from "../../strava.app.js";
+
+export default {
+  name: "Create Activity",
+  description: "Creates a manual activity for an athlete, requires activity:write scope. See `https://developers.strava.com/docs/reference/`",
```

Comment:
You can remove the "requires activity:write scope" section, since we've included that in our default authorization request